hecuba_py/src/hdict.py

```diff
@@ -416,7 +529,10 @@ def make_persistent(self, name):
                 log.warn("Error creating the StorageDict keyspace %s, %s", (query_keyspace), ex)
                 raise ex
 
-        all_columns = self._primary_keys + self._columns
+        if self._set_types is not None:
```

Comment:
It is needed because in the database table the set is just another column, like a simple attribute. So in self._columns it will be a simple attribute, but in self.__set_types we will know it is a set. We can change self._columns to { "set": { "attr": type } }, but then we will need to convert self._columns to only the attribute when using it.